File: 100 days of leetcode/Day18.py
class Solution(object):
    def largestAltitude(self, gain):
        """
        :type gain: List[int]
        :rtype: int
        """
        count = 0
        max = count

        for i in range(len(gain)):
            count += gain[i]
            if(count > max):
                max = count
        
        return max

File: 100 days of leetcode/Day18.java
class Solution {
    public int largestAltitude(int[] gain) {
        int count = 0, max = 0;

        for(int i = 0; i < gain.length; i++){
            count += gain[i];
            if(count > max){
                max = count;
            }
        }
        return max;
    }
}